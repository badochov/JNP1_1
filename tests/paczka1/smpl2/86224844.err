Error in line 5: jf4ln S467 7,3
Error in line 12: xvjd S680 87,1
Error in line 29: jf4ln S680 20,3
Error in line 37: qQhN8l S680 86,0
