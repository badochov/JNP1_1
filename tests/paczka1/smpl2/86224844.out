A629 S 18,3
a8tpJ A 318,6
S467 18,3
A629 318,6
A629 S 18,3
J8Xrq3e S 1,0
ZwIcymd8 A 10,9
a8tpJ A 318,6
jf4ln S 0,5
S467 19,3
A629 329,5
S680 0,5
A629 S 18,3
J8Xrq3e S 1,0
ZwIcymd8 A 10,9
a8tpJ A 318,6
jf4ln S 0,5
S467 19,3
A629 329,5
S680 0,5
A629 S 156,1
A629 329,5
